crates/dpapi-pdu/src/rpc/pdu.rs

```diff
@@ -573,12 +573,12 @@ impl DecodeWithContextOwned for PduData {
             packet_type => return Err(PduError::PduNotSupported(packet_type).into()),
         };
 
-        if !buf.is_empty() {
-            return Err(DecodeError::other(
-                "invalid PDU data",
-                "PDU header or PDU data is invalid: too much data to decode",
-            ));
-        }
+        // if !buf.is_empty() {
+        //     return Err(DecodeError::other(
+        //         "invalid PDU data",
+        //         "PDU header or PDU data is invalid: too much data to decode",
+        //     ));
+        // }
```

Comment:
Leftover?

Reply:
Yes, +
This PR still has a few places that need to be cleaned up 🙏 (this is why it is draft 😅)